pkg/instance/instance.go

```diff
@@ -133,7 +133,17 @@ func (ops V1DataEngineInstanceOps) InstanceCreate(req *rpc.InstanceCreateRequest
 	if err != nil {
 		return nil, grpcstatus.Error(grpccodes.Internal, errors.Wrapf(err, "failed to create ProcessManagerClient").Error())
 	}
-	defer pmClient.Close()
+
+	defer func() {
+		if closeErr := pmClient.Close(); closeErr != nil {
+			logrus.WithFields(logrus.Fields{
+				"name":            req.Spec.Name,
+				"type":            req.Spec.Type,
+				"dataEngine":      req.Spec.DataEngine,
+				"upgradeRequired": req.Spec.UpgradeRequired,
+			}).WithError(closeErr).Warn("Failed to close client")
```

Comment:
NIT: mention the client's name. e.g., `Failed to close ProcessManagerClient`. Similar below

Reply:
Added. Thank you.